src/App.vue

```diff
@@ -499,30 +489,30 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     window.addEventListener('resize', () => this.setWindowWidth(window.innerWidth))
 
     // listen for save error event
-    this.$root.$on('save-error-event', async error => {
-      // save errors/warnings
-      this.saveErrors = error?.response?.data?.errors || []
-      this.saveWarnings = error?.response?.data?.warnings || []
-
-      if (error?.response?.status === StatusCodes.PAYMENT_REQUIRED) {
-        // changes were saved if a 402 is received, so clear flag
-        this.setHaveChanges(false)
-        this.paymentErrorDialog = true
-      } else {
-        console.log('Save error =', error) // eslint-disable-line no-console
-        this.saveErrorDialog = true
-      }
-    })
-
-    this.$root.$on('name-request-invalid-error', async error => {
-      console.log('NR error during File and Pay =', error) // eslint-disable-line no-console
-      this.fileAndPayInvalidNameRequestDialog = true
-    })
-
-    this.$root.$on('name-request-retrieve-error', async () => {
-      console.log('Error while retrieving NR during File and Pay') // eslint-disable-line no-console
-      this.nameRequestInvalidErrorDialog = true
-    })
+    // this.$root.$on('save-error-event', async error => {
```

Comment:
Need to work on the alternative here (https://v3-migration.vuejs.org/breaking-changes/events-api.html#_2-x-syntax.